pepper-apis/local-support-environment/docker-compose.yml

```
@@ -0,0 +1,67 @@
+version: "3.9"
+services:
+  db:
+    image: mysql:5.7
+    environment:
+      MYSQL_USER: pepper
+      MYSQL_PASSWORD: pepper-dev
+      MYSQL_ROOT_PASSWORD: pepper-dev
+      TZ: "America/New_York"
```

This should be the UTC timezone
```suggestion
      TZ: "Etc/UTC"
--


[Etc/UTC](https://en.wikipedia.org/wiki/Coordinated_Universal_Time)"
```